dsDisplay.c

```diff
@@ -444,7 +444,6 @@ dsError_t dsGetEDIDBytes(intptr_t handle, unsigned char *edid, int *length)
 {
 	dsError_t ret = dsERR_NONE;
 	uint8_t buffer[128];
-	unsigned char *edidBuf = NULL;
 	size_t offset = 0;
 	int i, extensions = 0;
 	VDISPHandle_t *vDispHandle = (VDISPHandle_t *) handle;
```

Comment:
Missing validation of the args/buffers.
If any of these: handle and length is invalid(pointer cannot be NULL, handle should not be invalid) it will result in crash.